src/forms/inputs/dropdown-select.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { compose } from '../../utils'
+import { onOutsideClick, toggle, togglePropTypes } from '@launchpadlab/lp-hoc'
+import classnames from 'classnames'
+
+const propTypes = {
+  ...togglePropTypes('expanded'),
+  text: PropTypes.string,
+  selectedValues: PropTypes.arrayOf(PropTypes.string),
+  children: PropTypes.node,
+}
+
+const defaultProps = {
+  selectedValues: [],
+}
+
+// Wraps the `DropdownCheckboxGroup` component
```

Comment:
We might wanna move this to `helpers` since it's not publicly documented and also technically not a redux form input component.

Reply:
Yeah I was debating that as well. Will do